### PR TITLE
Exclude java ee imports

### PR DESCRIPTION
**Pull request by corneil**

### CODE REVIEW

File: spring-cloud-deployer-cloudfoundry/pom.xml [hunk 1]
```diff
@@ -43,6 +43,12 @@
 			<groupId>org.cloudfoundry</groupId>
 			<artifactId>cloudfoundry-client-reactor</artifactId>
 			<version>${cloudfoundry-java-lib.version}</version>
+			<exclusions>
```
Comment (onobc) [inline spring-cloud-deployer-cloudfoundry/pom.xml:46]: @corneil do you mind adding a well-known comment in each of these exclusions so we can easily find it later. 

How about `// @TODO boot3 jakarta` ?

**Review by onobc — CHANGES_REQUESTED** [review]
@corneil LGTM. One minor ask on adding comments.

File: spring-cloud-deployer-cloudfoundry/pom.xml (hunk 1 above)
Comment (corneil) [reply spring-cloud-deployer-cloudfoundry/pom.xml:48]: Updated

**Review by onobc — APPROVED** [review]
LGTM - thanks @corneil

**Pull request merged**